access/build.gradle

```diff
@@ -1,13 +1,17 @@
 plugins {
     id 'java-library'
     id 'maven-publish'
-    id 'signing'
     id 'checkstyle'
-    id 'org.quiltmc.gradle.licenser' version "${quiltmcLicenserVersion}"
+    id 'dev.yumi.gradle.licenser' version "${gradleLicenserVersion}"
```

Comment:
Hast du den Yumi Licenser mal getestet?

Reply:
Jap, funktioniert. Hat aber soweit ich gesehen habe in jeder Datei Fehler geworfen, weil die License sich am Anfang in Zeilenumbrüchen unterscheidet

Reply:
Kannst du in dem Branch mit  'applyLicenses' einfach mal alle Licences updaten und schauen ob das dann funktioniert? Dann sollten keine Fehler mehr kommen beim Check.